torchao/csrc/cuda/fp6_llm/utils_parallel_dequant.cuh

```diff
@@ -20,19 +20,23 @@
 
 #include <cuda.h>
 #include <cuda_fp16.h>
+// TODO: can cuda_bf16 be imported for SM75? How to guard against this? The guard below does not work outside of device code
```

Comment:
Where is bf16 used in host code?

Reply:
I'll try to hack around a bit more to see if I can make it work.

Reply:
It should work now. I think the problem was that `#include <cuda_bf16.h>` actually needs to be there for host code as well, so I changed it to `#if !defined(__CUDA_ARCH__) || __CUDA_ARCH__ >= 800`